module/stylesheets/components/_avatar.scss

```diff
@@ -0,0 +1,23 @@
+.avatar {
+  font-size: 32px;
+  display: flex;
+  height: 50px;
+  width: 50px;
+
+  img {
+    height: 50px;
```

Comment:
Move theses to variables. And try use "rem" instead "px"

Reply:
Will move this request to a new PR